<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>876</width>
    <height>703</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="pushButtonFilterSyncing">
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {	
	border:2px solid rgb(170, 170, 0);
	border-radius:5px;
	padding:5px;
}
QPushButton:checked {
	background-color:rgb(170, 170, 0);
}
</string>
          </property>
          <property name="text">
           <string>syncing</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonFilterChecking">
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {	
	border:2px solid rgb(255,255,0);
	border-radius:5px;
	padding:5px;
}
QPushButton:checked {
	background-color:rgb(255,255,0);
}
</string>
          </property>
          <property name="text">
           <string>checking</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonFilterFinished">
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {	
	border:2px solid rgb(0,255,0);
	border-radius:5px;
	padding:5px;
}
QPushButton:checked {
	background-color:rgb(0,255,0);
}
</string>
          </property>
          <property name="text">
           <string>finished</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonFilterFailed">
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {	
	border:2px solid rgb(255,0,0);
	border-radius:5px;
	padding:5px;
}
QPushButton:checked {
	background-color:rgb(255,0,0);
}
</string>
          </property>
          <property name="text">
           <string>failed</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListView" name="listView"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>876</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="geometry">
     <rect>
      <x>617</x>
      <y>406</y>
      <width>213</width>
      <height>108</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionexit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="geometry">
     <rect>
      <x>745</x>
      <y>406</y>
      <width>213</width>
      <height>72</height>
     </rect>
    </property>
    <property name="title">
     <string>帮助</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuDonate">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>213</width>
      <height>72</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="title">
     <string>捐赠</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
   <addaction name="menuDonate"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionexit">
   <property name="text">
    <string>exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
